content/en/tracing/guide/inferred-service-opt-in.md

```diff
@@ -48,19 +48,45 @@ Requirements:
 
 Set the following in your `datadog.yaml` [configuration file][5]:
 
-{{< code-block lang="yaml" filename="service.datadog.yaml" collapsible="true" >}}
+{{< code-block lang="yaml" filename="datadog.yaml" collapsible="true" >}}
 
 DD_APM_COMPUTE_STATS_BY_SPAN_KIND=true 
 DD_APM_PEER_TAGS_AGGREGATION=true
 DD_APM_PEER_TAGS='["_dd.base_service","amqp.destination","amqp.exchange","amqp.queue","aws.queue.name","bucketname","cassandra.cluster","cassandra.keyspace","db.cassandra.contact.points","db.couchbase.seed.nodes","db.hostname","db.instance","db.name","db.system","grpc.host","hazelcast.instance","hostname","http.host","messaging.destination","messaging.destination.name","messaging.kafka.bootstrap.servers","messaging.rabbitmq.exchange","messaging.system","mongodb.db","msmq.queue.path","net.peer.name","network.destination.name","peer.hostname","peer.service","queuename","rpc.service","rpc.system","server.address","streamname","tablename","topicname"]'
 
 {{< /code-block >}}
 
-### OpenTelemetry Collector Datadog Exporter configuration
+### OpenTelemetry Collector 
 
-Set the following in your `collector.yaml` [configuration file][6]:
-- `compute_stats_by_span_kind=true`
-- `peer_tags_aggregation=true`
+Minimum version recommended: opentelemetry-collector-contrib >= [v0.95.0][7].
+
+Example [collector.yaml][6].
+
+{{< code-block lang="yaml"  collapsible="true" >}}
+
+connectors:
+  datadog/connector:
+    traces:
+      compute_stats_by_span_kind: true
+      peer_tags_aggregation: true
+      peer_tags: ["_dd.base_service","amqp.destination","amqp.exchange","amqp.queue","aws.queue.name","bucketname","cassandra.cluster","db.cassandra.contact.points","db.couchbase.seed.nodes","db.hostname","db.instance","db.name","db.system","grpc.host","hazelcast.instance","hostname","http.host","messaging.destination","messaging.destination.name","messaging.kafka.bootstrap.servers","messaging.rabbitmq.exchange","messaging.system","mongodb.db","msmq.queue.path","net.peer.name","network.destination.name","peer.hostname","peer.service","queuename","rpc.service","rpc.system","server.address","streamname","tablename","topicname"]
+
+{{< /code-block >}}
+
+If your collector version is below [v0.95.0][7], you need to configure the exporter with the peer tags config:
+
+### Datadog Exporter configuration
```

Comment:
```suggestion
```
It's a little confusing to go to a new section here from the previous sentence, so I've removed the header and incorporated it into the previous sentence.